ckanext/repo/plugin.py

```diff
@@ -96,7 +98,10 @@ def _get_repo_info(repo_ref):
         log.error('Could not load module for repo {0}'.format(repo_ref))
         return None
 
-    cwd = os.path.join(os.path.dirname(module.__file__), cwd_dir)
+    if src_path:
+        cwd = os.path.join(os.path.dirname(module.__file__), src_path, repo_ref)
```

Comment:
If `src_path` is the absolute path why do we need to join `os.path.dirname(module.__file__)` first?